Makefile

```diff
@@ -58,6 +58,7 @@ ifneq ($(HAS_SERVER),)
 	fi; \
 
 	@echo Running golangci-lint
+	$(GO) install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.64.8
```

Comment:
That allows us to pin a version of golangci-lint. In the future, we can do that with https://tip.golang.org/doc/modules/managing-dependencies#tools.